OpenEdXMobile/src/test/java/org/edx/mobile/util/VersionTest.java

```diff
@@ -148,4 +148,35 @@ public void testCompareTo_withMorePreciseEarlierVersion_isGreaterThan() throws P
     public void testCompareTo_withMorePreciseLaterVersion_isLessThan() throws ParseException {
         assertThat(new Version("1")).isLessThan(new Version("1.0.1"));
     }
+
+    /**
+     * Verify that the {@link Version#isNMinorVersionsDiff(Version, int)} method returns true
+     * if passed another instance with greater or equal minor versions difference than the specified
+     * value.
+     */
+    @Test
+    public void testIsNMinorVersionsDiff_withGreaterOrEqualDiff_isTrue() throws ParseException {
+        final Version version = new Version("2.2.9");
+        assertThat(version.isNMinorVersionsDiff(new Version("2.0.1"), 2)).isEqualTo(true);
```

Comment:
i don't think so, difference between minor versions is 2 ( abs(2-0) >=2 ) . How should it be false ?